src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+
+    private List<Answer> answers = new ArrayList<>();
+
+    public Answers() {
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public List<Answer> of() {
+        return answers;
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public void delete(NsUser loginUser, List<DeleteHistory> deleteHistories) throws CannotDeleteException {
+        if (answers.stream()
+                .anyMatch(answer -> !answer.isOwner(loginUser))) {
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+        }
+
+        answers.forEach(answer -> answer.delete(deleteHistories));
```

Comment:
컬렉션을 매개변수로 전달해 자료구조의 제어권을 넘기고, 상대방이 값을 변경하도록 하는건 너무 외부에 의존적인 것 같습니다. 차라리 다음과 같이 history를 반환받아 collection으로 만드는게 낫지 않을까요? 

```suggestion
        answers.stream().map(Answer::delete).collect(toList());
```